buildSrc/src/main/groovy/com/uber/okbuck/core/dependency/DependencyUtils.groovy

```diff
@@ -13,19 +13,19 @@ final class DependencyUtils {
 
     private DependencyUtils() {}
 
-    static Set<Configuration> useful(Project project, Set<String> configurations) {
-        Set<Configuration> useful = new HashSet<>()
-        configurations.each { String configName ->
-            try {
-                useful.add(project.configurations.getByName(configName))
-            } catch (UnknownConfigurationException ignored) {}
-        }
+    static Configuration useful(Project project, String configuration) {
+        try {
+            Configuration config = project.configurations.getByName(configuration)
+            return useful(config)
+        } catch (UnknownConfigurationException ignored) {}
```

Comment:
return null in the catch block and change `ignored` to `e`

Reply:
done